src/panels/developer-tools/info/integrations-card.ts

```diff
@@ -65,19 +65,22 @@ class IntegrationsCard extends LitElement {
                               Documentation
                             </a>
                           </td>
-                          ${!manifest.is_built_in
-                            ? ""
-                            : html`
+                          ${manifest.is_built_in || manifest.issue_tracker
+                            ? html`
                                 <td>
                                   <a
-                                    href=${integrationIssuesUrl(domain)}
+                                    href=${integrationIssuesUrl(
+                                      domain,
+                                      manifest
+                                    )}
```

Comment:
Maybe?
```suggestion
                                    href=${manifest.issue_tracker || integrationIssuesUrl(domain)}
```

Reply:
That works too, but with that format, it will be easier to forget about custom integrations in the future, so I prefer to keep it as is if possible.

Reply:
Then apply my suggestion to `integrationIssuesUrl`